visible-label-in-acc-name: editorial updates (#1469)
act-rules/act-rules.github.io: _rules/visible-label-in-accessible-name-2ee8b8.md

## Background

> This rule applies to elements with a [widget role][] that [support name from content][supports name from content]. This includes the following: `button`, `checkbox`, `gridcell`, `link`, `menuitem`, `menuitemcheckbox`, `menuitemradio`, `option`, `radio`, `searchbox`, `switch`, `tab`, `treeitem`.

Reviewer: "This includes" sounds like the list is not exhaustive. AFAICR, this list is exhaustive, so let's take the opportunity to
```suggestion
This rule applies to elements with a [widget role][] that [support name from content][supports name from content], namely: `button`, `checkbox`, `gridcell`, `link`, `menuitem`, `menuitemcheckbox`, `menuitemradio`, `option`, `radio`, `searchbox`, `switch`, `tab`, `treeitem`.
```

Author: I don't really agree with that. Much prefer the current proposal.

> The understanding document of [2.5.3 Label in Name][understand253] use the term "symbolic text characters" to refer to a type of [non-text content][] that text characters as symbols, such as using "x" to mean "close". Because there is no definition for symbolic text characters, this rule uses non-text content instead.

Reviewer: ```suggestion
The understanding document of [2.5.3 Label in Name][understand253] use the term "symbolic text characters" to refer to a type of [non-text content][] that uses text characters as symbols, such as using "x" to mean "close". Because there is no definition for symbolic text characters, this rule uses non-text content instead.
```

> The understanding document of [2.5.3 Label in Name][understand253] use the term "symbolic text characters" to refer to a type of [non-text content][] that text characters as symbols, such as using "x" to mean "close". Because there is no definition for symbolic text characters, this rule uses non-text content instead.

Reviewer: I'm not a fan of the last sentence. Especially "because there is no definition of foo" looks both a bit lazy (what don't we write the definition if we need it) and not entirely true (the Understanding document of 1.4.5 has an almost definition of this, and even the UC of 2.5.3 is giving a lot of details on why this mean).
Additionally, the Expectation is already saying "except for characters used to express non-text content" which is almost a definition of "symbolic characters"…

I do agree that explaining the difference in wording and giving example makes sense.
What about replacing the last sentence by something like
> This rule considers them as "characters expressing non-text content".

or something in this line?

Maybe also adding that "characters expressing non-text content" also includes emojis (at least for some definition of characters, but emojis are Unicode codepoints so I say they qualify as characters). That would point out that our name is a bit more generic than "symbolic characters" and justify why we use this instead.

Author: Thanks for the suggestions. I've taken another stab at it. Let me know!

> The understanding document of [2.5.3 Label in Name][understand253] use the term "symbolic text characters" to refer to a type of [non-text content][] that uses text characters as symbols, such as using "x" to mean "close". This rule considers them as "characters expressing non-text content". Unicode emojis are another example of non-text content, although these are not "symbolic text characters".

Reviewer: ```suggestion
The understanding document of [2.5.3 Label in Name][understand253] use the term "symbolic text characters" to refer to a type of [non-text content][] that uses text characters as symbols, such as using "x" to mean "close". This rule considers them as "characters expressing non-text content". Unicode emojis are another example of characters expressing non-text content, although these are not "symbolic text characters".
```
